public/js/thirdpartyservices.js

```diff
@@ -101,6 +101,14 @@ var thirdPartyServicesJsonArray = [
     link: {
       privacy: 'https://privacy.oath.com',
       terms: 'https://developer.yahoo.com/flurry/legal-privacy/terms-service/flurry-analytics-terms-service.html'
+    },
+    {
+    name: 'ACRA',
+    model: 'ACRA',
+    acra: false,
+    logo: 'images/acra.png',
+    link: {
+      privacy: 'https://github.com/ACRA/acra/',
```

Comment:
This doesn't link to the privacy policy of ACRA, but the GitHub repo. You need to provide the URL to the privacy policy of ACRA service. 

Take a look at links for other services that already exist in this JSON.

Reply:
I did not find a privacy policy of ACRA. I think that they do not have one. 😨

Reply:
well, in that case, adding this to the web app will be misleading. Eventually, the web app generates a Privacy policy that has these links. People who use this web app don't bother to check these links and assume it is correct. If this PR is merged this will end up in a lot of apps with a link that doesn't point to a privacy policy. 

Maybe what you should do is request on their repo to include one and then add the link.

I will keep the PR open until that happens, so your effort is not wasted.